scala/src/main/scala/smile/validation/Normalize_Mutual_Index.scala

```diff
@@ -0,0 +1,67 @@
+import scala.math.{max, log, sqrt}
+
+/**
+ * @author Beck Gaël
+ */
+object Externals_Indexes
```

Comment:
In general, the file name should not contain underscore _ and the class name is better same as the file name. It is just a matter of style. Better be consistent with the rest of library.

More importantly, why is this in smile.validation package? For fundamental math functions like this, I suggest too add it to smile.math.Math. Also the Java users can leverage it too.